src/keylessBackup/KeylessBackupPhoneCodeInput.tsx

```diff
@@ -77,6 +126,33 @@ const styles = StyleSheet.create({
     color: colors.primary,
     ...typeScale.labelSemiBoldMedium,
   },
+  bottomSheetTitle: {
+    ...typeScale.titleSmall,
+    textAlign: 'center',
+  },
+  bottomSheetView: {
+    paddingHorizontal: 24,
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  bottomSheetBody: {
+    ...typeScale.bodyMedium,
+    textAlign: 'center',
+  },
+  buttonContainer: {
+    flexDirection: 'row',
+    marginTop: 37,
+    marginBottom: 9,
+    alignItems: 'center',
+    justifyContent: 'center',
+    gap: 96,
```

Comment:
Needed this to correctly space the two buttons, not sure if there is a better way so open to suggestions!